terraform/etl/38-aws-glue-job-parking.tf

```
@@ -1257,3 +1257,26 @@ module "parking_permit_street_stress" {
     "--environment"         = var.environment
   }
 }
+# MRB 14-08-2023 Job created 
+module "parking_permit_street_stress_with_cpz" {
+  source                         = "../modules/aws-glue-job"
+  is_live_environment            = local.is_live_environment
+  is_production_environment      = local.is_production_environment
+  department                     = module.department_parking_data_source
+  job_name                       = "${local.short_identifier_prefix}parking_permit_street_stress_with_cpz"
+  helper_module_key              = data.aws_s3_object.helpers.key
+  pydeequ_zip_key                = data.aws_s3_object.pydeequ.key
+  spark_ui_output_storage_id     = module.spark_ui_output_storage_data_source.bucket_id
+  script_name                    = "parking_permit_street_stress"
```

just checking, should this be parking_permit_street_stress_with_cpz?
```suggestion
  script_name                    = "parking_permit_street_stress_with_cpz"
```